code/game/objects/items/stunbaton.dm

```diff
@@ -283,3 +350,74 @@
 	desc = "A cost-effective, mass-produced, tactical stun prod."
 	preload_cell_type = /obj/item/stock_parts/cell/high/plus // comes with a cell
 	color = "#aeb08c" // super tactical
+
+
+///TESTING PROCS
+
+GLOBAL_VAR_INIT(batons_instant, FALSE)
+GLOBAL_VAR_INIT(batons_stam, FALSE)
+GLOBAL_VAR_INIT(batons_normal, TRUE)
+GLOBAL_VAR_INIT(batons_seconly, FALSE)
+GLOBAL_VAR_INIT(batons_cooldown, TRUE)
+
+/datum/admins/proc/cmd_batons_instant_stun()
+	set category = "Batons"
+	set name = "Toggle Batons Instant Stun"
+	if(!check_rights(R_DEV))
+		return
+	GLOB.batons_instant = !GLOB.batons_instant
+	to_chat(usr, "Instant Batons are now set to [GLOB.batons_instant]")
+		
+
+/datum/admins/proc/cmd_batons_stamina()
+	set category = "Batons"
+	set name = "Toggle Baton 1/2 hit (Stam)"
+	if(!check_rights(R_DEV))
+		return
+	/// TRUE IS 1 HIT
+	GLOB.batons_stam = !GLOB.batons_stam
+	to_chat(usr, "Batons are now set to 1 hit TRUE/FALSE - [GLOB.batons_stam]")
+
+/datum/admins/proc/cmd_batons_normal()
+	set category = "Batons"
+	set name = "Toggle Batons to Normal or Modified" ///Safety Net
+	if(!check_rights(R_DEV))
+		return
+	GLOB.batons_normal = !GLOB.batons_normal
+	if(GLOB.batons_normal == FALSE)
+		GLOB.batons_normal = TRUE
+		GLOB.batons_stam = FALSE
+		GLOB.batons_seconly = FALSE
+		GLOB.batons_cooldown = FALSE
+		GLOB.batons_instant = FALSE
+		to_chat(usr, "Batons are now RESET")
+	else
+		GLOB.batons_normal = FALSE
+		to_chat(usr, "Batons can now be modified")
+
+/datum/admins/proc/cmd_batons_seconly()
+	set category = "Batons"
+	set name = "Toggle Batons Sec Only"
+	if(!check_rights(R_DEV))
+		return
+	GLOB.batons_seconly = !GLOB.batons_seconly
+	to_chat(usr, "Batons are now set to Sec only - TRUE/FALSE [GLOB.batons_seconly]")
+
+/datum/admins/proc/cmd_batons_cooldown()
+	set category = "Batons"
+	set name = "Toggle Batons Cooldown"
```

Comment:
Why is this just disable cooldowns? I thought we wanted to test different amounts of cooldown